test/lib/tasks/seeds_test.rb

```diff
@@ -70,13 +70,6 @@ class PulpProxyTest < SeedsTest
     end
   end
 
-  class PermissionsTest < SeedsTest
-    test "Make sure katello permissions got created exist" do
-      seed
-      assert Permission.pluck(:resource_type).grep(/Katello/).present?
-    end
-  end
```

Comment:
I think it may be the permissions get created dynamically on app start-up only, and the database gets cleaned between tests.  I think it's fine to remove this, as they're not seeded anymore.